committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+# SIG Governance Template
+
+## Goals
+
+The following document outlines a template governance structure for
+SIGs to follow.  Its goals are to provide the necessary structure
+for SIGs to self organize and self govern.
+
+This document is focused on:
+
+- Outlining organizational responsibilities
+- Outlining organizational roles
+- Outlining processes and tools
+
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of project leadership
+- The process for making technical decisions
+- The process for ensuring continued test health
+
+## How to use this template
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+**Note:** There are 2 charter template versions:
+
+- a super-lite TL;DR version that only provides a abstract structure.
+- a detailed version that attempts to concretely define a high
+  degree of structure.
+
+SIGs may either choose to start with the TL;DR charter and add
+structure from the detailed version, or to start with the detailed
+version and remove structure that does not provide direct value.
+
+**Note:** Specific responsibilities towards the project are required
+for some SIGs and should be included in the charter - e.g. ensuring
+test health.
+
+FAQ:
+
+- [Why is it so long?](#why-is-it-so-long)
+- [Why so much structure?](#why-so-much-structure)
+- [Why use mailing lists for tracking roles?](#why-use-mailing-lists-for-tracking-roles)
+- [Why track membership as a special role?](#why-track-membership-as-a-special-role)
+- [How do the results from the survey fit in?](#how-do-the-results-from-the-survey-fit-in)
+
+---
+
+# SIG ~~Foo~~ Governance TL;DR Version
+
+## Roles
+
+- Project Leads
+  - Run operations and processes governing the SIG
+  - Outgoing Project Leads may propose new ones -
+  - Number: 2-3
+- Technical Leads
+  - Escalation for technical discussions
+  - Outgoing Technical Leads may propose new ones -
+  - Number: 2-7
+- Members
+  - Maintain health of the project
+  - Build new functionality
+
+## Processes
+
+- Agreeing upon decisions
+  - Send for lazy consensus - send to the googlegroup with RFC or KEP in
```

Comment:
Lazy consensus implies anyone can block progress. I suggest some flavor of modified lazy consensus seeking, which falls back to a vote if consensus isn't achieved.

Example:
https://github.com/kubernetes/steering/blob/master/changes.md

Whether or not there is a vote, there also needs to be a notion of who is a participant vs. who is a decider:
https://stumblingabout.com/tag/oarp/

In SIG Arch today we discussed the idea of formalizing more roles within SIGs, similar to the release team, in order to clarify project needs, clarify role expectations and boundaries, provide recognition, and encourage more participation by providing status to people doing the work. So "contributors", whether code contributors, code reviewers, or people filling other roles would have standing for making decisions within the SIG.